cms/djangoapps/contentstore/views/component.py

```diff
@@ -64,7 +65,7 @@
         name for name, class_ in XBlock.load_classes()
         # TODO MRI-5684 - remove this exclusion
         # once ambra_case_list class is deleted.
-        if name != "ambra_case_list"
+        # if name != "ambra_case_list"
```

Comment:
Should these be deleted instead of commented?

Reply:
Whoops, that was unintentional. There is a separate branch that removes this that can be merged once we've gotten rid of the case list xblock: #51